### PR TITLE
Theme integrated and some other changes

### PR DESCRIPTION
**Pull request by VatsalMewada**

### CODE REVIEW

File: app/controllers/opening_jobs_controller.rb [hunk 1]
```diff
@@ -39,7 +41,7 @@ def interested_people
   end
 
   def jobs_list
-    @jobs = OpeningJob.all
+    @jobs = OpeningJob.all.where(open: true)
```
Comment (swati-bacancy) [inline app/controllers/opening_jobs_controller.rb:44]:
```suggestion
    @jobs = OpeningJobl.where(open: true)
```
also create one scope for open jobs

File: app/views/charges/_companies_list.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  <% @subscriptions.each do |subscription| %>
+    <tr>
+      <td title="Company details"><%= link_to subscription.company.name, company_path(subscription.company), style: "color:black;" %></td>
+      <td title ="Subscribed? "><%= subscribed(subscription) %></td>
+      <td title="Days to complete trial"><%= days_left(subscription) %></td>
+      <td title ="Subscription date"><%= subscription&.subscription_date %></td>
+    </tr>
+  <% end %>
+<% else %>
+  <span style="color: red; font-weight: bold">***Data not available</span>
```
Comment (swati-bacancy) [inline app/views/charges/_companies_list.html.erb:11]: please create class for css.

File: config/routes.rb [hunk 3]
```diff
@@ -24,13 +24,17 @@
...
+        get 'interested_people'
+        get 'open_close_post'
+      end
     end
   end
 
   resources :technologies
   resources :super_admins do
     collection do
+      resources :charges, only: :index
```
Comment (swati-bacancy) [inline config/routes.rb:37]: please add all the resource before collection

**Comment by VatsalMewada** [comment]
Branch already fetched and merged

**Pull request closed without merging**